tests/cbtest/gtest/test_uzfs.cc

```diff
@@ -114,6 +116,87 @@ uzfs_mgmt_conn_list_count(struct uzfs_mgmt_conn_list *list)
 	return count;
 }
 
+extern int epollfd;
+TEST(uZFS, asyncTaskProps) {
+
+	async_task_t *arg;
+	uzfs_mgmt_conn_t *conn;
+
+
+	conn = SLIST_FIRST(&uzfs_mgmt_conns);
+
+	epollfd = epoll_create1(0);
+	if (epollfd < 0) {
+		perror("epoll_create1");
+		zk_thread_exit();
+		return;
+	}
+
+	uzfs_zinfo_take_refcnt(zinfo, B_TRUE);
+	arg = (async_task_t *)kmem_zalloc(sizeof (*arg), KM_SLEEP);
```

Comment:
a function may be good by passing these values

Reply:
Done.